packages/react-next/src/components/Layer/Layer.test.tsx

```diff
@@ -8,7 +8,7 @@ import { safeCreate } from '@uifabric/test-utilities';
 const ReactDOM = require('react-dom');
 
 const testEvents: string[] = (
-  'click contextmenu doubleclick drag dragend dragenter dragleave dragover dragstart drop ' +
+  'click contextmenu double click drag dragend dragenter dragleave dragover dragstart drop ' +
```

Comment:
This is a literal event name and shouldn't be changed.
```suggestion
  'click contextmenu doubleclick drag dragend dragenter dragleave dragover dragstart drop ' +
```